src/pages/PullRequestPage/PullCoverage/routes/IndirectChangesTab/IndirectChangedFiles/IndirectChangedFiles.spec.tsx

```diff
@@ -565,7 +565,7 @@ describe('IndirectChangedFiles', () => {
           expect.objectContaining({
             filters: expect.objectContaining({
               ordering: {
-                direction: 'DESC',
+                direction: 'ASC',
```

Comment:
this was same as previous expect, when it should be checking the other direction

Reply:
Nice catch!